assets/src/edit-story/utils/elementsFromPoint.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+function elementsFromPoint(x, y) {
+  const elements = [],
```

Comment:
Ouch. This is very painful. Let's check in again offline if we can avoid `elementFromPoint` at all. I think we don't quite yet know the touch situation. Could it be possible that a target might still be possible here?

If not - I think we may need to switch to a different system where one the dragging starts, we'd ask all targets for their frame dimensions and order. At least then we could possibly optimize this code by reducing the number of possible matches?